### PR TITLE
Refactor jaeger parts

### PR DESCRIPTION
**Pull request by johscheuer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
 # GoDays Demo
 
-Each step of the demo can be built using `docker-compose build --build-arg step=4`. Then run the demo: `docker-compose up`
+Each step of the demo can be built using `docker-compose build --build-arg step=4`. Then run the demo: `docker-compose up -d`
```
Comment (maxbischoff) [inline README.md:3]:
```suggestion
Each step of the demo can be built using `docker-compose build --build-arg step=$STEP`. Then run the demo: `docker-compose up -d`
```
Replace step number with placeholder?

File: step_1/README.md [hunk 2]
```diff
@@ -5,6 +5,5 @@ Neither the backend nor the frontend send any spans to jaeger. You can access th
 ## Run this step
 
 ```sh
-docker-compose build --build-arg step=1
-docker-compose up
+docker-compose build up -d
```
Comment (maxbischoff) [inline step_1/README.md]: Missing build-arg?

**Review by maxbischoff — CHANGES_REQUESTED** [review]

File: step_1/README.md (hunk 2 above)
Comment (johscheuer) [reply step_1/README.md]: Nope the complete command makes no sense

File: README.md (hunk 1 above)
Comment (johscheuer) [reply README.md:3]: ok

**Review by maxbischoff — APPROVED** [review]

**Pull request merged**